src/ressources/customers/GroupChipField.tsx

```diff
@@ -0,0 +1,24 @@
+import { Chip } from "@mui/material";
+import { useRecordContext } from "react-admin";
+
+export const GroupChipField = ({ source }: { source: string }) => {
+  const record = useRecordContext();
+
+  return record[source]
+    ? record[source].map((item: string) => {
+        formatString(item);
```

Comment:
what is the purpose of this line?

Reply:
Thank you for pointing out this useless code.